### PR TITLE
add Sustainable Living chat

### PR DESCRIPTION
**Pull request by nicwaller**
## What

add "Sustainable Living" (https://chat.stackexchange.com/rooms/7310/sustainable-living) to the no-freeze list

## Why

It was suggested in chat, here: https://chat.stackexchange.com/transcript/message/61176987#61176987

## How

Just [follow the README](https://github.com/tripleee/sloshy/blob/master/README.md#configuration)

### CODE REVIEW

File: sloshy.yaml [hunk 1]
```diff
@@ -109,6 +109,9 @@ servers:
     - contact: oleg-valter (15810379)
       name: "V-eye"
       id: 240030
+    - contact: Nic (64473)
```
Comment (nicwaller) [inline sloshy.yaml:112]: My network profile: https://stackexchange.com/users/64473/nic

File: sloshy.yaml [hunk 2]
```diff
@@ -109,6 +109,9 @@ servers:
...
+      name: "Sustainable Living"
+      id: 7310
```
Comment (nicwaller) [inline sloshy.yaml:114]: The chat room: https://chat.stackexchange.com/rooms/7310/sustainable-living

**Review by nicwaller — COMMENTED** [review]
Adding PR comments to confirm details are correct.

**Comment by tripleee** [comment]
The failure is expected; see #19

Would be nice to have a proper test for this though ...

File: sloshy.yaml (hunk 2 above)
Comment (tripleee) [reply sloshy.yaml:114]: Shouldn't the room be added under the `chat.stackexchange.com` server then, not the `chat.stackoverflow.com` one?

Comment (tripleee) [reply sloshy.yaml:114]: (Also, tangentially, adding a room between Oleg's two rooms is mildly disturbing the file's organization; but if the entry has to be moved anyway, this is quite tangential.)

**Comment by tripleee** [comment]
Sorry for taking a few days to get back to you; I'll see if I could fix this up myself, but feel free to push changes if I don't seem to be able to merge soon. (In the middle of a forced OS upgrade, so somewhat distracted.)

**Comment by tripleee** [comment]
FWIW, success indicator: https://chat.stackoverflow.com/transcript/message/54622147#54622147

**Pull request merged**